draft-labiod-rats-attester-groups.md

```diff
@@ -105,43 +105,62 @@ The dynamic nature of an Attester Group allows for the flexibility to tailor gro
+
+Use Case 3: AI computing cluster
+
+Context:The AI computing cluster is a composite computing environment composed of a group of computing nodes/chips on which a computing task is executed.
+A user or an application/large model provider needs to verify the integrity of the collected measurement/evidence information from the composite computing environment.
+
+
+Challenge: The cluster may contain heterogeneous trusted roots, and the composition may be dynamically updated. Repeated attestation is not efficient and can be very expensive.
+
+Approach: We can consider a large group of attesters or a set of group attesters. A group attester corresponding to the computing cluster of the execution of a task may be dynamically created or canceled according to the requirements of the computing task. The remote attestation server/verifier verifies the collected measurement information for the composite computing environment. Remote group attestation should hide the complexity of back-end computing node interaction from customers. Generally, a master node in the group is responsible for communicating with the client, responding to the remote attestation challenge request of the client, collecting measurement information of all the group nodes as a whole, and sending the measurement information to the remote attestation server for appraisal.
+
+When all computing nodes/chips in a computing attester group are provided by the same vendor or deployed by the same cloud vendor, a unified and centralized dedicated hardware root of trust can be deployed (e.g., hardware security chip, centralized hardware DIE, or BMC) to offload important security functions (secure storage, security monitoring, etc.) to this independent module. The trusted boot and other related measurement information of the group are securely stored on the unified root of trust. During the remote attestation process, the master node of the group collects measurement from this module, and the unified root of trust signs all the measurement. If a single root of trust manages multiple chips, a single point of failure (such as malicious intrusion and system breakdown) of the root of trust affects the security of the entire attester group managed by the root of trust. The unified trusted root should support distributed and pooled design. Multiple roots of trust may work together to enhance overall security and reliability.
+
+In heterogeneous interconnection scenarios where all computing nodes and chips in a computing cluster are provided by different vendors, a unified root of trust may not be deployed. During the group remote attestation process, the master node needs to communicate with each group node to collect its own measurement information. The information is signed by the private key of each node. The master node collects the information, packs the information, and sends it to the remote attestation server for appraisal.
```

Comment:
```suggestion
In heterogeneous interconnection scenarios where all computing nodes and chips in a computing cluster are provided by different vendors, it might not be possible to deploy a unified root of trust. During the group remote attestation procedure, the master node needs to communicate with each group node to collect its individual evidence claims. The node evidence is signed by the private attestation key of each node. The master node collects the information, packs the information, and sends it to the remote verifier for appraisal.
```